SwrveSDK/SDK/Talk/SwrveMessageController.m

```diff
@@ -868,16 +865,22 @@ -(SwrveConversation*)getConversationForEvent:(NSString *)event
         
         // If QA enabled, send message selection information
         if(self.qaUser != nil) {
-            [self.qaUser trigger:event withConversation:result withReason:campaignReasons];
+            [self.qaUser trigger:eventName withConversation:result withReason:campaignReasons];
         }
     }
     
     if (result == nil) {
-        DebugLog(@"Not showing message: no candidate messages for %@", event);
+        DebugLog(@"Not showing message: no candidate messages for %@", eventName);
     }
     return result;
 }
 
+-(SwrveConversation*)getConversationForEvent:(NSString *)event
```

Comment:
both getConversationForEvent and findConversationForEvent should be named the same